qiskit/ignis/verification/accreditation/qotp.py

```diff
@@ -0,0 +1,250 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+# pylint: disable= no-member
+"""
+Applies a Quantum One Time Pad (QOTP)
+
+This is essentially the same protocol as used in
+randomized compiling, but follows the methods in
+
+https://iopscience.iop.org/article/10.1088/1367-2630/ab4fd6
+"""
+import numpy as np
+from qiskit import QuantumCircuit
+from qiskit.converters.circuit_to_dag import circuit_to_dag
+from qiskit.converters.dag_to_circuit import dag_to_circuit
+from qiskit.compiler import transpile
+
+
+def layer_parser(circ, twoqubitgate='cx', coupling_map=None):
+    """
+    Tranforms general circuits into a nice form for a qotp.
+        Args:
+            circ (QuantumCircuit): A generic quantum circuit
+            twoqubitgate (string): a flag as to which 2 qubit
```

Comment:
The documentation string should mention allowed values for two_qubit_gate and the default value (i think just "cx" or "cz" at the moment).